site/class-1-bar/index.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
@sophiemei Whoops, looks like your JSON is broken! (you can see the highlighting below going awry).

In JSON, keys (such as date and food) need "quotes" around them.

On GitHub, you can click the pencil icon on the top-right of a file to edit them, and you can do that here too.

Could you do that?